src/Draft/Draft.js

```diff
@@ -1,46 +1,127 @@
+import Grid from '@material-ui/core/Grid';
+import List from '@material-ui/core/List';
+import { Hidden } from '@material-ui/core';
+
+const GridContainer = styled(Grid)({
+  height: '100vh',
+  width: '100vw',
+  color: '#fff',
+  padding: '1em'
+});
+
+const AllTeamsContainer = styled(Grid)({
+  height: '100%',
+  width: '100%',
+  overflow: 'hidden'
+});
+
+const MidGrid = styled(Grid)({
+  height: '100%',
+  width: '100%',
+  overflow: 'hidden',
+  display: 'flex',
+  justifyContent: 'space-between',
+  padding: '1em 1em 1em 2em'
+});
 
 class Draft extends Component {
   constructor(props) {
-    super(props)
-    this.state = { teams: [], players: [], myTeam: null }
+    super(props);
+    this.state = {
+      teams: [
+        'team1',
+        'team2',
+        'team3',
+        'team4',
+        'team5',
+        'team6',
+        'team7',
+        'team8',
+        'team9',
+        'team10',
+        'team11',
+        'team12'
+      ],
+      players: [],
+      myTeam: null,
+      currentPick: 0
+    };
   }
 
   componentDidMount() {
-    this.subscription = new Subscription('/draft', data => {
-      this.setState({
-        myTeam: data.teams.find(team => {
-          return team.roster.find(p => p._key === this.props.user._key)
-        }),
-        teams: data.teams,
-        players: data.players
-      })
-    })
+    // this.subscription = new Subscription('/draft', data => {
+    //   this.setState({
+    //     myTeam: data.teams.find(team => {
+    //       return team.roster.find(p => p._key === this.props.user._key)
+    //     }),
+    //     teams: data.teams,
+    //     players: data.players
+    //   })
+    // })
   }
 
-  componentWillUnmount() {
-    this.subscription && this.subscription.end()
-  }
+  // componentWillUnmount() {
+  //   this.subscription && this.subscription.end()
+  // }
 
   render() {
+    // return (
+    //   <div style={{ color: 'white' }}>
+    //     {this.state.myTeam ? <MyTeam user={this.props.user} team={this.state.myTeam} /> : ''}
+    //     <MediaQuery minDeviceWidth={1224}>
+    //       <TeamList user={this.props.user} teams={this.state.teams} />
+    //     </MediaQuery>
+    //     <CurrentBid user={this.props.user} />
+    //     <PlayerList user={this.props.user} players={this.state.players} />
+    //   </div>
+    // )
     return (
-      <div style={{ color: 'white' }}>
-        {this.state.myTeam ? <MyTeam user={this.props.user} team={this.state.myTeam} /> : ''}
-        <MediaQuery minDeviceWidth={1224}>
-          <TeamList user={this.props.user} teams={this.state.teams} />
-        </MediaQuery>
-        <CurrentBid user={this.props.user} />
-        <PlayerList user={this.props.user} players={this.state.players} />
-      </div>
-    )
+      <GridContainer className="container" container spacing={8}>
+        <AllTeamsContainer className="allTeams" item xs={2}>
+          <h1>All Teams</h1>
+          <List component="div" className="teamsList">
```

Comment:
List shouldn't wrap a single component.  If you want to use List, use it in TeamList